src/test/java/org/example/CalendarManagement/api/validator/ValidateMeetingDateTimeTestDetails.java

```diff
@@ -0,0 +1,58 @@
+package org.example.CalendarManagement.api.validator;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+class ValidateMeetingDateTimeTestDetails {
```

Comment:
class name should end with Test, otherwise i don't think surefire plugin will run this..

Reply:
yes